contributing.md

```diff
@@ -135,7 +135,11 @@ Once built, load it in the browser of your choice:
 			<p>Or you can use <a href="https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference#web-ext_run"><code>web-ext run</code></a> to have Firefox automatically load and reload it:</p>
 			<pre>npx web-ext run --source-dir=distribution</pre>
 			<p>Recommended arguments:</p>
-			<pre>npx web-ext run --source-dir=distribution --firefox-profile=refined-github --keep-profile-changes --start-url=github.com</pre>
+			<ul>
+				<li><code>--firefox-profile=refined-github</code></li>
+				<li><code>--keep-profile-changes</code></li>
+				<li><code>--start-url=github.com</code></li>
+			</ul>
```

Comment:
I liked it better in a single command. You can make it multiline as:

```suggestion

			```
			npx web-ext run \
			--source-dir=distribution \
			--firefox-profile=refined-github \
			--keep-profile-changes
			--start-url=github.com
			```
```

Also I'd like to see this for Chrome as well. Perhaps it should be a separate table. Like:

## Loading into the browser

Once built, load it in the browser of your choice.

### Load with [`web-ext`](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference#web-ext_run)

< table with web-ext commands>

### Load manually

< current table >